lib/reporters/html/partials/aggregations.hbs

```diff
@@ -53,4 +51,8 @@
             </div>
         </div>
     </div>
+{{/each}}
+{{#if parent.name}}
```

Comment:
This is not good, we should not have two related parts of the template in separate if blocks